guide/03-the-gis/cloning-content.ipynb

```diff
@@ -0,0 +1,2051 @@
+{
```

Comment:
<p>quotation marks missing on the right side -</p><p>should be <code>target = GIS(profile="your_online_admin_profile", verify_cert=False)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Fixed it!</p>